bucketeer/src/test/kotlin/io/bucketeer/sdk/android/internal/evaluation/EvaluationInteractorTest.kt

```diff
@@ -118,11 +120,12 @@ class EvaluationInteractorTest {
       ),
     )
 
-    val interactorWithEmptyFeatureTag = componentWithEmptyFeatureTag.evaluationInteractor
-    assertThat(interactorWithEmptyFeatureTag.currentEvaluationsId).isEqualTo("")
-
-    interactorWithEmptyFeatureTag.currentEvaluationsId = "should_be_clear"
-    assertThat(interactorWithEmptyFeatureTag.currentEvaluationsId).isEqualTo("should_be_clear")
+    componentBuildWithEmptyFeatureTag.evaluationInteractor.also {
```

Comment:
You don't use it parameter, so please remove `also`

same for other `also` in tests.

Reply:
it because, we need access `component. evaluationInteractor ` as its a lazy property. when creating , the logic for init will trigger (compare & save the feature_tag...) 
I did like this because there is a yellow warning about unused variable and I want to remove it.
Let me know if you have suggestion for this.

Reply:
I see.
Then, we should declare a wrapper extension method for `also` to give it a more descriptive name, like, `prepareDependencyAndRun`

Reply:
changed in https://github.com/bucketeer-io/android-client-sdk/pull/89/commits/d369910da1ec7ad27e5259d9f67b950ce3344d30

Reply:
updated in https://github.com/bucketeer-io/android-client-sdk/pull/89/commits/d369910da1ec7ad27e5259d9f67b950ce3344d30